docs/installing.md

```diff
@@ -129,3 +124,74 @@ It is also recommended to have:
 
 Additional resources may be required depending on the workloads you plan to run.
 
+### Installation via .deb Package
+
+Add the Rancher Desktop repository and install Rancher Desktop with:
+
+```
+curl https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/Release.key | sudo apt-key add -
```

Comment:
On two separate systems, an old one and a new 20.04.3 LTS, when I try to run this command, I get the error message `gpg: no valid OpenPGP data found.`  Looking at the output of the curl command is a 404. Highlights:
```
<title>Resource is no longer available!</title>
...
<body>
<h1>Resource is no longer available!</h1>
<p>


    The requested URL is no longer available on this server and there is no
    forwarding address.



    If you followed a link from a foreign page, please contact the
    author of this page.



</p>
<p>
If you think this is a server error, please contact
the <a href="mailto:webmaster@opensuse.org">webmaster</a>.

</p>

<h2>Error 404</h2>
```